tests/e2e/framework/mason.go

```diff
@@ -18,7 +18,7 @@ import (
 	"fmt"
 	"io/ioutil"
 
-	"gopkg.in/yaml.v2"
+	yaml "gopkg.in/yaml.v2"
```

Comment:
For this and other similar - did you use an automated tool ? Better to keep cosmetic changes separate.

Reply:
I used goimports. It was because make lint was complaining about it. It is better not to change and commit with the lint warnings?